frontend/src/components/DemoModal/DemoModal.tsx

```diff
@@ -30,7 +30,25 @@ export const DemoModal = () => {
 		if (!(await formStore.validate())) {
 			return
 		}
-		if (email.indexOf('@gmail.') !== -1) {
+		const badEmailStrings = [
+			'gmail',
+			'yahoo',
+			'hotmail',
+			'work',
+			'mysite',
+			'outlook',
+			'thanks.com'
```

Comment:
looks like , is missing here
```suggestion
			'thanks.com',
```